src/access/acp.ts

```diff
@@ -19,17 +19,26 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import { WithAccessibleAcr } from "../acp/acp";
-import { getAcrPolicyUrlAll, getPolicyUrlAll } from "../acp/control";
-import { internal_getAcr } from "../acp/control.internal";
-import { getPolicyAll } from "../acp/policy";
-import { getRuleAll } from "../acp/rule";
 import { acp } from "../constants";
-import { WithResourceInfo } from "../interfaces";
 import { getSourceIri } from "../resource/resource";
-import { getSolidDataset } from "../resource/solidDataset";
-import { getUrlAll } from "../thing/get";
-import { asUrl, getThing } from "../thing/thing";
+import { getThing } from "../thing/thing";
+import { hasAccessibleAcr, WithAccessibleAcr, WithAcp } from "../acp/acp";
+import {
+  AccessControlResource,
+  getAcrPolicyUrlAll,
+  getPolicyUrlAll,
+} from "../acp/control";
+import { internal_getAcr } from "../acp/control.internal";
+import { getAllowModes, getDenyModes, getPolicy, Policy } from "../acp/policy";
+import {
+  getForbiddenRuleurlAll,
```

Comment:
Typo:
```suggestion
  getForbiddenRuleUrlAll,
```

Reply:
Oh whoops, this is not actually a typo in the import, but in the original function. Which means changing this will break the API - so I'll do that in a separate PR and make sure it's backwards-compatible, so it can be separately reviewed.